sqladmin/templates/details.html

```diff
@@ -18,18 +18,19 @@ <h3 class="card-title">{{ model_admin.pk_column.name }}: {{ model_admin.get_attr
             {% for name, attr in model_admin._details_attrs %}
             <tr>
               <td>{{ name }}</td>
+              {% set value, formatted_value = model_admin.get_detail_value(model, attr) %}
               {% if (name, attr) in model_admin._details_relations %}
-              {% if is_iterable( model_admin.get_list_value(model, attr) ) %}
```

Comment:
I seems to be a typo of `get_detail_value(...)`, right?

Reply:
Yes.